components/applications-service/pkg/ingester/v1/ingest.go

```diff
@@ -16,7 +16,7 @@ import (
 )
 
 const (
-	numWorkers       = 50
+	numWorkers       = 25
```

Comment:
🤔 this might be correct? I noticed that running the integration tests while all of A2 was running, I ran out of database connections... probably need to make it configurable eventually.